src/client/components/CvPdfRender.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+
+export function CvPdfRender() {
+    const [cvPdfFile, setCvPdfFile] = useState()
+    const { id } = useParams();
+    useEffect(() => {
+        (async () => {
+            const response = await fetch('/api/cv');
```

Comment:
```suggestion
            const response = await fetch('/api/cv')
               .json()
               .find(cv => cv.fk_user_id === id);
```

Reply:
if i change the code like this am getting this error (CvPdfRender.js:11 Uncaught (in promise) TypeError: fetch(...).json is not a function)